libkbfs/key_manager.go

```diff
@@ -450,21 +450,41 @@ func (km *KeyManagerStandard) Rekey(ctx context.Context, md *RootMetadata, promp
 			return false, nil, err
 		}
 		resolvedInfo := resolvedHandle.ConflictInfo()
-		if latestHandle.ConflictInfo != resolvedInfo {
-			km.log.CDebugf(ctx, "handle for %s is conflicted",
-				handle.GetCanonicalPath())
-			if resolvedInfo != nil {
-				// ConflictInfo is inconsistent.
+		if resolvedInfo == nil {
```

Comment:
This block might work better as a function on `TlfHandleExtension`, maybe something like `ShouldOverride(other *TlfHandleExtension) (bool, err)`?  `Rekey` is already pretty long.